packages/app/src/KubernetesDashboardPage.tsx

```diff
@@ -259,7 +260,7 @@ export const InfraPodsStatusTable = ({
             row["arrayElement(ResourceAttributes, 'k8s.namespace.name')"],
           node: row["arrayElement(ResourceAttributes, 'k8s.node.name')"],
           restarts: row['last_value(k8s.container.restarts)'],
-          uptime: row['sum(k8s.pod.uptime)'],
+          uptime: row['undefined(k8s.pod.uptime)'],
```

Comment:
Its kinda funny but technically no aggFn, thoughts? @dhable

Reply:
It does look strange expressed like that but probably good enough for now. Maybe we need to alias something like `any` to mean undefined in the string?

Reply:
Yeah we probably need to revisit the alias here. Just want to point it out